webdriver/tests/bidi/network/remove_intercept/remove_intercept.py

```diff
@@ -0,0 +1,90 @@
+# META: timeout=long
+
+import asyncio
+
+import pytest
+from webdriver.bidi.modules.script import ScriptEvaluateResultException
+
+from .. import (
+    assert_before_request_sent_event,
+    assert_response_event,
+)
+
+PAGE_EMPTY_HTML = "/webdriver/tests/bidi/network/support/empty.html"
+PAGE_EMPTY_TEXT = "/webdriver/tests/bidi/network/support/empty.txt"
+PAGE_OTHER_TEXT = "/webdriver/tests/bidi/network/support/other.txt"
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("phase", ["beforeRequestSent", "responseStarted"])
+async def test_remove_intercept(
+    bidi_session, wait_for_event, url, setup_network_test, fetch, phase
+):
+    network_events = await setup_network_test(
+        events=[
+            "network.beforeRequestSent",
+            "network.responseStarted",
+            "network.responseCompleted",
+        ]
+    )
+    before_request_sent_events = network_events["network.beforeRequestSent"]
+    response_started_events = network_events["network.responseStarted"]
+    response_completed_events = network_events["network.responseCompleted"]
+
+    text_url = url(PAGE_EMPTY_TEXT)
+    intercept = await bidi_session.network.add_intercept(
+        phases=[phase],
+        url_patterns=[{"type": "string", "pattern": text_url}],
+    )
+
+    assert isinstance(intercept, str)
```

Comment:
We could be more precise here and assert it is an UUID.

Chromium impl uses a regex for that: https://github.com/GoogleChromeLabs/chromium-bidi/blob/60071ea69bd6614e3b721949727544dc69a45907/tests/test_helpers.py#L147

Reply:
Oh good point. I should actually test that in add_intercept.py actually, with a "payload" test. I see some other modules in wdspec use `uuid.UUID` to assert if a UUID is valid. If that's ok I'll try using this as well.